entrypoint.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/ash
+
+set -euo pipefail
+
+chown -R pypiserver:pypiserver /data
+# Set the setgid bit so anything added here gets associated with the
+# pypiserver group
+# chmod g+s /data/packages
```

Comment:
Is this commented out intentionally?

Reply:
Right, I will remove those comments.

It's not necessary to set permissions like that, I found.